app/code/Magento/Checkout/view/frontend/web/js/view/minicart.js

```diff
@@ -94,17 +94,17 @@ define([
                 this.isLoading(addToCartCalls > 0);
                 sidebarInitialized = false;
                 this.update(updatedCart);
+
+                if (cartData()['website_id'] !== window.checkout.websiteId) {
```

Comment:
AFAIK this file is using not at checkout page only, so window.checkout will not be available

Reply:
I checked your comment.
window.checkout is available on home page and category pages.
![2018-03-26 09-51-13](https://user-images.githubusercontent.com/10788564/37891075-9fcdf9c0-30db-11e8-9d4f-ed470e4525ca.png)
Also, I want to tell you, that logic "cartData()['website_id'] !== window.checkout.websiteId" was earlier in Magento.
It was fixed it to remove excessed request.

Reply:
Will review deeper little bit later

Reply:
I can confirm that window.checkout property exist at any page where present minicart

Reply:
Hello, @sidolov 
Please check and approve this PR.
Thanks for advance.